packages/graphql/src/resolvers/User.js

```diff
@@ -145,6 +146,20 @@ async function reviews(user, { first = 10, after }) {
       event.returnValues.ipfsHash,
       event
     )
+
+    if (review.reviewer && !review.reviewer.id) {
+      console.log(
+        '============== Non-nullable error in User.reviews resolver! ============='
+      )
+      console.log('User.id', review.reviewer.id)
+      console.log('path: ', apolloPathToString(info.path))
+      console.log('returnType: ', info.returnType)
+      console.log('operation: ', info.operation.operation)
+      console.log(
+        '========================================================================='
+      )
+    }
+
     if (review.rating) {
```

Comment:
maybe lets add a check here that reviewer.id is also available before the node gets added?